tests/tests_test_workflow/test_integ_workflow/integ_test/test_local_test_cluster.py

```diff
@@ -151,54 +147,7 @@ def test_wait_for_service_cluster_unavailable(self, mock_test_recorder, *mocks):
             )
         self.assertEqual(str(err.exception), "Cluster is not available after 10 attempts")
 
-    @patch(
-        "test_workflow.integ_test.local_test_cluster.psutil.Process",
-        side_effect=__mock_process,
-    )
-    @patch("test_workflow.integ_test.local_test_cluster.subprocess.Popen.wait")
-    @patch("test_workflow.integ_test.local_test_cluster.subprocess.Popen.terminate")
-    @patch("test_workflow.integ_test.local_test_cluster.logging", return_value=MagicMock())
-    def test_terminate_process(self, mock_logging, mock_terminate, mock_wait, mock_process):
-        self.local_test_cluster.stdout = None
-        self.local_test_cluster.stderr = None
-        self.local_test_cluster.process = self.process
+    @patch.object(Process, 'terminate', return_value=("1", "2", "3"))
```

Comment:
Is this the number of termination or the seconds count for termination in return value?

Reply:
Just provide some return values for the mock function in order for the test to continue without complaining.